src/com/projectkorra/projectkorra/earthbending/metal/Extraction.java

```diff
@@ -59,42 +63,33 @@ public void progress() {
 		Material type;
 		ItemStack item;
 
-		switch (this.originBlock.getType()) {
-		case IRON_ORE:
+		switch (this.originBlock.getType().name()) {
+		case "IRON_ORE":
+		case "DEEPSLATE_IRON_ORE":
 			type = Material.STONE;
-			item = new ItemStack(Material.RAW_IRON, this.getAmount() * 2);
+			item = new ItemStack(is117 ? Material.getMaterial("RAW_IRON") : Material.IRON_INGOT, this.getAmount() * 2);
 			break;
-		case GOLD_ORE:
+		case "GOLD_ORE":
+		case "DEEPSLATE_GOLD_ORE":
 			type = Material.STONE;
-			item = new ItemStack(Material.RAW_GOLD, this.getAmount() * 2);
+			item = new ItemStack(is117 ? Material.getMaterial("RAW_GOLD") : Material.GOLD_INGOT, this.getAmount() * 2);
 			break;
-		case NETHER_QUARTZ_ORE:
+		case "NETHER_QUARTZ_ORE":
 			type = Material.NETHERRACK;
 			item = new ItemStack(Material.QUARTZ, this.getAmount());
 			break;
-		case NETHER_GOLD_ORE:
+		case "NETHER_GOLD_ORE":
 			type = Material.NETHERRACK;
 			item = new ItemStack(Material.GOLD_NUGGET, this.getAmount() * 6);

```

Comment:
I haven't touched the amounts, it was already like that. Perhaps ask @Aztlon

Reply:
oh so this is why i kept getting emails from this pr. it's multiplied because it drops gold nuggets instead of ingots but i kinda forgot why it does that.